src/lib/babel.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @param {import('../configure').Options} options
+ */
+export function babelConfig(options) {
```

Comment:
This file is basically the same as babel-loader.js except I separated the config part from the loader part so I could re-use it in rollup.